relevanceai/operations/dr/base.py

```diff
@@ -32,14 +32,22 @@ def fit(self, *args, **kw):
     def transform(self, *args, **kw):
         raise NotImplementedError
 
-    def transform_documents(self, vector_field: str, documents: List[Dict]):
-        vectors = self.get_field_across_documents(vector_field, documents)
-        return self.transform(documents)
+    def transform_documents(self, vector_fields: List[str], documents: List[Dict]):
+        vectors = np.array(
+            self.get_fields_across_documents(
+                vector_fields, documents, missing_treatment="skip"
+            )
+        )
+        vectors = vectors.reshape(-1, vectors.shape[1] * vectors.shape[2])
+        return self.transform(vectors)
 
-    def fit_documents(self, vector_field: str, documents: List[Dict]):
-        vectors = self.get_field_across_documents(
-            vector_field, documents, missing_treatment="skip"
+    def fit_documents(self, vector_fields: List[str], documents: List[Dict]):
+        vectors = np.array(
+            self.get_fields_across_documents(
+                vector_fields, documents, missing_treatment="skip"
+            )
         )
+        vectors = vectors.reshape(-1, vectors.shape[1] * vectors.shape[2])
```

Comment:
You're flattening the vectors? For DR?

Reply:
Yeah, the get_fields_across_documents produces a 3D vector of (len(docs), len(vector_fields), ? or len(vector), if both vector same length))